modules/ROOT/pages/dw-arrays-functions-some.adoc

```diff
@@ -8,10 +8,9 @@ endif::[]
 [[some1]]
 == some&#40;Array<T&#62;, &#40;T&#41; &#45;&#62; Boolean&#41;: Boolean
 
-Returns `true` if an element in the array matches the specified condition.
+Returns `true` if at least one element in the array matches the specified condition.
 
-
-The function stops iterating after the first match to an element in the array.
+The function stops iterating after the first element is found that matches the condition.
```

Comment:
> The function stops iterating after the first element that matches the condition is found.

Reply:
done in adoc file only, will update dw repo separately

Reply:
done in dw repo